<ui version="4.0" >
 <class>UserScript</class>
 <widget class="QWidget" name="UserScript" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QTextEdit" name="programEdit" />
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>4</number>
      </property>
      <property name="leftMargin" >
       <number>4</number>
      </property>
      <property name="topMargin" >
       <number>4</number>
      </property>
      <property name="rightMargin" >
       <number>4</number>
      </property>
      <property name="bottomMargin" >
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="statusLabel" >
        <property name="text" >
         <string>Unknown</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="compileButton" >
     <property name="text" >
      <string>Compile</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
